lara-typescript/src/interactive-api-host/attachments-api/attachments-manager.ts

```diff
@@ -79,13 +79,17 @@ export class AttachmentsManager {
     const publicPath = this.tokenServiceClient.getPublicS3Path(folderResource, `${this.sessionId}/${name}`);
     const url = await this.getSignedUrl(folder, "putObject", { Key: publicPath, ContentType, expiresIn });
     // returns the writable url and the information required to read it
-    return [url, { folder, publicPath }];
+    return [url, { folder, publicPath, contentType: ContentType }];
   }
 
   public getSignedReadUrl(attachmentInfo: IReadableAttachmentInfo, options?: ISignedReadUrlOptions) {
-    const { publicPath, folder } = attachmentInfo;
+    const { publicPath, folder, contentType } = attachmentInfo;
     const { expiresIn = kDefaultReadExpirationSec } = options || {};
-    return this.getSignedUrl(folder, "getObject", { Key: publicPath, expiresIn });
+    return this.getSignedUrl(folder, "getObject", {
+      Key: publicPath,
+      expiresIn,
+      ContentType: contentType
```

Comment:
I'm not sure, but I don't see how the ContentType would actually be used here.
During a post it is needed to allow the client to send the content type header. But during a get I'd assume the client isn't sending a header, and I don't think S3 will do something to enforce the file type matches the content type of the signature.

Reply:
I've removed the content type from the signed read url request.  One note - the lower `getSignedUrl` request is defaulting the content type to `text/plain`, but again this doesn't seem to matter on read urls.